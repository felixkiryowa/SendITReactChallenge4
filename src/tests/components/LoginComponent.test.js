import React from 'react';
import renderer from 'react-test-renderer';
import LoginComponent from '../../components/Auth/Login';

describe('Sample Articles Component', () => {
    let mountApp;
    const props = {
        handleSubmit: jest.fn(),
        handlePasswordChange: jest.fn(),
        handleUsernameChange: jest.fn(),
        username: 'Roy22',
        password: 'user123',
        errors: {
            
        },
    };
    beforeEach(() => {
        mountApp = renderer.create(<LoginComponent {...props} />);
    });

    it('Should render without crashing', () => {
        expect(mountApp).toMatchSnapshot();
    });
});
