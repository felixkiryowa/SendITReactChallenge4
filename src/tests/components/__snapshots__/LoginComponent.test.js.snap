// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Articles Component Should render without crashing 1`] = `
<div
  className="card card-container"
>
  <img
    className="profile-img-card"
    id="profile-img"
    src="//ssl.gstatic.com/accounts/ui/avatar_2x.png"
  />
  <p
    className="profile-name-card"
    id="profile-name"
  />
  <form
    className="form-signin"
    onSubmit={[MockFunction]}
  >
    <span
      className="reauth-email"
      id="reauth-email"
    />
    <input
      autoFocus={true}
      className="form-control UsernameInput"
      id="inputEmail"
      name="username"
      onChange={[MockFunction]}
      placeholder="Enter Username"
      required={true}
      type="text"
      value="Roy22"
    />
    <br />
    <input
      className="form-control PasswordInput"
      id="inputEmail"
      name="password"
      onChange={[MockFunction]}
      placeholder="Enter Password"
      required={true}
      type="password"
      value="user123"
    />
    <button
      className="btn btn-lg btn-primary btn-block btn-signin login-btn"
      type="submit"
    >
      Sign in
    </button>
  </form>
  <a
    className="forgot-password"
    href="#"
  >
    Forgot the password?
  </a>
</div>
`;
